lock-api/src/main/java/com/palantir/lock/SimpleHeldLocksToken.java

```diff
@@ -15,13 +15,12 @@
  */
 package com.palantir.lock;
 
-import java.io.Serializable;
 import java.math.BigInteger;
 
 import com.fasterxml.jackson.annotation.JsonProperty;
 import com.google.common.base.Preconditions;
 
-public class SimpleHeldLocksToken implements Serializable {
+public class SimpleHeldLocksToken implements LockToken {
```

Comment:
HeldLocksToken is also a LockToken (I also made SimpleHeldLocksToken work for completeness)